bindings/java/src/main/com/apple/foundationdb/FutureResult.java

```diff
@@ -33,5 +38,12 @@
 		return FutureResult_get(cPtr);
 	}
 
+	@Override
+	protected void postMarshal(byte[] value){
```

Comment:
Should this also call `super.postMarshal` to ensure the future is closed?

Reply:
yes, thanks